golem-api-grpc/proto/golem/apidefinition/api_definition.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package golem.apidefinition;
+
+import "golem/template/template_id.proto";
+
+message ApiDefinition {
+  oneof definition {
+    HttpApiDefinition http = 1;
+  }
+}
+
+message ApiDefinitionList {
+  oneof definitions {
+    HttpApiDefinitionList http = 1;
+  }
+}
+
+message HttpApiDefinitionList {
+  repeated HttpApiDefinition definitions = 1;
+}
+
+message HttpApiDefinition {
+  ApiDefinitionId id = 1;
+  string version = 2;
+  repeated HttpRoute routes = 3;
+}
```

Comment:
just one idea, probably better design could be 
```proto
message ApiDefinition {
  ApiDefinitionId id = 1;
  string version = 2;
  oneof {
     HttpApiDefinition http = 1;
  }
}

message HttpApiDefinition {
  repeated HttpRoute routes = 1;
}


message GrpcApiDefinition {
  . ..
}

to have common attributes, and then specific ones


```

Reply:
damn it. you're right 😆 

I will fix it